src/transforms/posthtml.js

```diff
@@ -26,7 +26,7 @@ async function transform(asset) {
 
 async function getConfig(asset) {
   let config =
-    (await asset.getPackage()).posthtml ||
+    ((await asset.getPackage()) || {}).posthtml ||
```

Comment:
This is not the correct fix, there is an option you can pass to asset.getConfig, that already prevents this behaviour. Better to use this than to reimplement it in every place.

Reply:
You mean: js```await asset.getConfig([ 
   '.posthtmlrc', 
   '.posthtmlrc.js', 
   'posthtml.config.js' 
], {
  'packageKey': 'posthtml'
})```? I tried, but this doesn't work. 
The problem is not in asset.getConfig(). The error comes from line 29:32, this line

Reply:
Yes I mean that, the original asset.getPackage becomes obsolete when using getConfig options. You can always check out how asset.getConfig works to understand

Reply:
Ok, thanks!